src/components/PrevRate.js

```diff
@@ -1,6 +1,6 @@
 import React, { PureComponent } from "react";
 import { array } from "prop-types";
-import { View, TouchableOpacity, ScrollView, FlatList } from "react-native";
+import { View, TouchableOpacity, FlatList } from "react-native";
```

Comment:
Wondering if this is related to this PR.

Reply:
Wonder not. It was left by me in the last PR. Only right that it be removed since it is something minor